src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -683,6 +699,12 @@ private String getProviderName() {
      * security_guide_jdk_providers JDK Providers} document for information
      * on the transformation defaults used by JDK providers.
      *
+     * @implNote
```

Comment:
You also need a similar implNote for `getInstance(String, String)`.

Reply:
Yes, good catch, thanks!